packages/viewer/src/lib/walk-mode/model.ts

```diff
@@ -56,7 +56,7 @@ export class WalkModeModel {
   };
 
   private configuration: ViewerWalkModeConfiguration = {
-    teleportHeightScalar: 8.5,
+    teleportHeightPercentage: 8.5,
```

Comment:
Do you want to update this to 11.75?

Reply:
Ah, yes. Unsaved change when I pushed this up 🙃